enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -349,27 +349,68 @@ func (c *COWStore) ReadAt(p []byte, off int64) (int, error) {
 	chunkOffset := c.chunkStartOffset(off)
 	n := 0
 	for len(p) > 0 {
```

Comment:
Not introduced in your PR, but do we want to return an error if we try to read bytes beyond the max size of the store?

Reply:
good catch, will send a separate one for that